src/Compilers/CSharp/Test/Emit2/CodeGen/CodeGenSpanBasedStringConcatTests.cs

```diff
@@ -1479,22 +1504,19 @@ public void M()
               // Code size       50 (0x32)
               .maxstack  3
               .locals init (char V_0,
-                            char V_1,
-                            char V_2)
-              IL_0000:  ldc.i4.s   97
-              IL_0002:  stloc.0
-              IL_0003:  ldloca.s   V_0
-              IL_0005:  newobj     "System.ReadOnlySpan<char>..ctor(ref readonly char)"
+                            char V_1)
+              IL_0000:  ldstr      "a"
+              IL_0005:  call       "System.ReadOnlySpan<char> string.op_Implicit(string)"
```

Comment:
This flip is going from making a ROS from a single character to loading a constant single-character string and converting it to a ROS. Talking with Stephen Toub, we expect it to be a neutral change; I can make the new code use the character-based path, but that seemed like more code to maintain for no real benefit.